Doc/Doxyfile

```diff
@@ -733,7 +733,7 @@ WARN_LOGFILE           =
 # spaces.
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = mainpage.md ../ ../manager ../splines ../samplePDF ../OscClass ../mcmc ../Diagnostics ../Diagnostics/Diagnostics_utils ../covariance Plots/
+INPUT                  = mainpage.md ../ ../manager ../splines ../samplePDF ../OscClass ../mcmc ../Diagnostics ../plotting ../plotting/plottingUtils ../Diagnostics/Diagnostics_utils ../covariance Plots/
```

Comment:
Can you update this as well
https://github.com/mach3-software/MaCh3/blob/develop/cmake/Templates/Doxyfile.in

I know it may look confusing.

Both doxyfiles are identical except
https://github.com/mach3-software/MaCh3/blob/9b319d291c2bfba67fb3b1abdc6ab25367cc9cb9/cmake/Templates/Doxyfile.in#L41
.in take Software version and should replace actual Foxyfile